tt1-blocks/experimental-theme.json

```diff
@@ -208,6 +208,19 @@
 				"lineHeight": "var(--wp--custom--line-height--body)"
 			}
 		},
+		"core/button": {
+			"border": {
+				"radius": "0px"
+			},
+			"color": {
+				"background": "var(--wp--preset--color--gray)",
+				"text": "var(--wp--preset--color--green)"
+			},
+			"typography": {
+				"fontSize": "22.5px",
```

Comment:
Should we use one of the preset font sizes? Or add this to them?

Reply:
I had that debate in my head and went back and forth.

I finally landed on the above to achieve design parity with the original and didn't add it as a preset font size, again to achieve feature parity with TwentyTwentyOne (as it [doesn't provide that one)](https://github.com/WordPress/twentytwentyone/blob/trunk/functions.php#L146).

So my vote is no to both.

Reply:
What about the same question for font-weight?

Reply:
That one I'm impartial about.  Personally I don't really see much value in making it a variable but I can't argue that it shouldn't be.

Reply:
👍 ✅